README.md

```diff
@@ -7,6 +7,8 @@ This project very much under active development, and is currently not as reusabl
 
 ## Configuring for Development
 
+Make sure an instance of elasticsearch is available at `http://elasticsearch.local:9200/`. If you are running theonion/onion-services you can add `elasticsearch.local` to your `/etc/hosts` line that points at that docker dev machine.
+
```

Comment:
@benghaziboy was not a fan.

We were able to us the `onion-services`  ES at `localhost:9200` by forwarding the port in VirtualBox.

I'm in favor of leaning on /etc/hosts but let's see if we can get consensus on this.

It is noteworthy that the stock ES in hombrew is at an incompatible version where onion-services ES is version locked.